src/qps.cpp

```diff
@@ -782,7 +782,7 @@ void Qps::menu_custom()
     }
     else
     {
-        field_win = new FieldSelect(procview);
+        field_win = new FieldSelect(this, procview);
```

Comment:
Please revert this one. Users usually want to see what happens on the fly.

And please put all instances of `QWidget *parent` at the end as `QWidget *parent = nullptr` for more flexibility.

Reply:
Done.